tests/users/test_api_list_users.py

```diff
@@ -50,6 +50,19 @@ def test_list_users_api_resource_auth(self):
         self.assertEqual(200, actual_response.status_code)
         self.assertEqual(expected_response, json.loads(actual_response.data))
 
+    def test_search_user_api_resource_auth(self):
+        auth_header = get_test_request_header(self.admin_user.id)
+        print("verified user: %s"%self.verified_user.name)
```

Comment:
Can you remove the print statements? I'm sure you used them just for testing, right?

Reply:
done!